src/ripple/server/impl/Port.cpp

```diff
@@ -217,6 +217,21 @@ parse_Port (ParsedPort& port, Section const& section, std::ostream& log)
     set(port.ssl_cert, "ssl_cert", section);
     set(port.ssl_chain, "ssl_chain", section);
     set(port.ssl_ciphers, "ssl_ciphers", section);
+
+    port.pmd_options.server_enable =
+        section.value_or("permessage_deflate", false);
+    port.pmd_options.client_max_window_bits =
+        section.value_or("client_max_window_bits", 15);
+    port.pmd_options.server_max_window_bits =
+        section.value_or("server_max_window_bits", 15);
+    port.pmd_options.client_no_context_takeover =
+        section.value_or("client_no_context_takeover", false);
+    port.pmd_options.server_no_context_takeover =
+        section.value_or("server_no_context_takeover", false);
+    port.pmd_options.compLevel =
+        section.value_or("compress_level", 3);
+    port.pmd_options.memLevel =
+        section.value_or("memory_level", 4);
```

Comment:
There are two inconsistencies between these statements and the `struct permessage_deflate` declaration in beast/websocket/option.hpp, so I would like to double check that these aren't accidents:

1.  Is `client_enable` missing?
2. Should the default for `compLevel` be 8?